src/components/Button/Button.styles.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+import { pallete } from "../../misc/pallete";
+
+const buttonSize = {
```

Comment:
takiego consta warto dorzucić do pliku ze stałymi, bo teraz siedzi razem ze stylami. Z racji, że jest to często używany obiekt to możesz go wrzucić do katalogu 'src' i np stworzyć taki globalny plik const.js na takie wartości, które są używane w całej aplikacji